src/controllers/user.controller.js

```diff
@@ -0,0 +1,88 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../models/user.js';
+import { emailService } from '../services/email.service.js';
+import { userService } from '../services/user.service.js';
+import bcrypt from 'bcrypt';
+
+const getAllActivated = async (req, res) => {
+  const users = await userService.getAllActivated();
+
+  res.send(users.map(userService.normalize));
+};
+
+const getProfile = async (req, res) => {
+  const user = await User.findByPk(req.user.id);
+
+  res.send(user);
+};
+
+const changeName = async (req, res) => {
+  const { newName } = req.body;
+
+  if (!newName) {
+    throw res.send(ApiError.badRequest('New name is required'));
+  }
+
+  const user = await User.findByPk(req.user.id);
+  user.name = newName;
+  await user.save();
+
+  res.send({message: 'Message sent successfully', user});
+};
+
+const changePassword = async (req, res) => {
+  const { oldPassword, newPassword, confirmPassword } = req.body;
+
+  if (!oldPassword || !newPassword || !confirmPassword) {
+    throw ApiError.badRequest('All fields are required');
+  }
+
+  const user = await User.findByPk(req.user.id);
+  const isMatch = await bcrypt.compare(oldPassword, user.password);
+
+  if (!isMatch) {
+    throw ApiError.badRequest('Old and new passwords do not match');
```

Comment:
The error message 'Old and new passwords do not match' is misleading. It should indicate that the old password is incorrect. Consider changing it to 'Old password is incorrect'.